osidb/models.py

```diff
@@ -88,10 +88,6 @@ def search_helper(
 class FlawHistoryManager(ACLMixinManager):
     """flaw history manager"""
 
-    def get_queryset(self):
-        """define base queryset for retrieving flaws"""
-        return super().get_queryset()
-
```

Comment:
this should be removed in the first commit, you can do this with an interactive rebase [1].

[1] https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#_changing_multiple

Reply:
makes sense, moved to the first commit.